lib/tizen_sdk.dart

```diff
@@ -268,6 +263,34 @@ class TizenSdk {
       return version;
     }
 
+    if (profile == 'tizen') {
+      if (versionToDouble(apiVersion) < 8.0) {
+        throwToolExit(
+            'The $apiVersion version is not supported by the tizen profile.');
+      }
+    } else if (profile == 'common') {
+      if (versionToDouble(apiVersion) >= 8.0) {
+        // Note: Starting with Tizen 8.0, the unified "tizen" profile is used.
+        profile = 'tizen';
+      } else {
+        if (arch == 'x86') {
+          // Note: The TV emulator use mobile rootstrap.
+          profile = 'mobile';
+        } else {
+          // Note: The headless profile is not supported.
+          profile = 'iot-headed';
+        }
+      }
+    } else if (profile == 'tv') {
+      // Note: The tv-samsung and the tv rootstrap is not publicly available.
+      profile = 'tv-samsung';
+    } else if (profile == 'mobile') {
+      if (versionToDouble(apiVersion) >= 8.0) {
+        throwToolExit(
+            'The $apiVersion version is not supported by the mobile profile.');
```

Comment:
Can we just use the tizen rootstrap as a fallback in this case?

Reply:
In fact, this case does not occur unless user build the 8.0 application manually. (` flutter-tizen build tpk -pmobile`)
Because there is no 8.0 mobile device and emulator. However, to prevent the above build command from failing, I changed it to use `tizen` rootstrap.